PlantPal/PlantPal/WelcomeScreen.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ContentView.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 07/07/2024.
+//
+
+import SwiftUI
+// MARK: - VIEW
+
+struct WelcomeScreen: View {
+    // MARK: - Properties
+    
+    @EnvironmentObject private var viewModel: WelcomeScreenViewModel
+    
+    // MARK: - BODY
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                
+                Color("BackgroundColor")
+                    .ignoresSafeArea()
+                
+                VStack {
+                    Image("Plant")
+                    
+                    Text("PlantPal AI")
```

Comment:
Wpisywane z palca teksty mają kilka dużych minusów. Dwa podstawowe to brak możliwości przetłumaczenia aplikacji na inne języki i zajmowanie dużo miejsca.